### ISSUE TITLE
Error with Deploying Sample from Windows-rs Repo

### ISSUE THREAD

**Issue by ashenparikh** [microsoft/windows-rs#1853]
Hi! I attempted to deploy one of the samples (sample_com_uri) running the command: cargo run --bin sample_com_uri. I did install the Visual Studio C++ build tools for windows 11 + English Pack.
I got the error below. The extra operand is the 3rd one. The output of the command run /usr/bin/link has more than 20 + operands. 

I'm new to the repo so the fix is probably something simple. Any suggestions would be appreciated

Error:
/usr/bin/link: extra operand 'C:\\Users\\Ashen Parikh\\Desktop\\windows-rs\\target\\debug\\build\\quote-5224ac6119b73cb0\\build_script_build-5224ac6119b73cb0.build_script_build.61469ee2-cgu.1.rcgu.o'

**Comment by riverar** [comment]
Not sure what `/usr/bin/link` is pointing to, sounds like you're using a WSL or Linux environment?

**Comment by ashenparikh** [comment]
Hi @riverar, for this repo, does the commands work only in Visual Studio since Visual Studio contains the C++ Build tools. I'm using Visual Studio Code currently

**Comment by ashenparikh** [comment]
I got different output in the cmd terminal of Visual Studio code saying the following: 

   Compiling windows_x86_64_msvc v0.38.0 (C:\Users\Ashen Parikh\Desktop\windows-rs\crates\targets\x86_64_msvc)
error: linker `link.exe` not found
  |
  = note: program not found

note: the msvc targets depend on the msvc linker but `link.exe` was not found

note: please ensure that VS 2013, VS 2015, VS 2017, VS 2019 or VS 2022 was installed with the Visual C++ option

**Comment by riverar** [comment]
Sounds like your environment is not quite set up for Rust development. Check out https://docs.microsoft.com/windows/dev-environment/rust/setup for a step by step.

**Comment by ashenparikh** [comment]
Yup! I'm adding the dependencies and testing it once the install is complete

**Comment by ashenparikh** [comment]
I followed the documentation and installed the suggested dependencies. I can use the windows crate and the bindings inside. I tested it with the posted code in the readme and it was successfully run, which is great news.

The original error is there: with the linker can't be found when the (cargo run --bin sample_com_uri) command is run. I'm reading more about rust linkage after compilation so I can understand the command dependencies better.

Just wanted to share an update

**Comment by ashenparikh** [comment]
I downloaded the Microsoft C++ Build Tools build dependencies: .NET desktop development, Desktop development with C++, and Universal Windows Platform development. 

Using Visual Studio Code cmd terminal I ran both the succesful and unsuccessful run.
1. For the succesful run, I created a new project directory and then ran the code from readme. I ran cargo run. It imported the bindings and the code ran. 
2. For the failed run with the linkner error, I navigated to the windows-rs project. Inside of that I ran the (cargo run --bin sample_com_uri).
